apps/system/lockscreen/raptor/lockscreen.esp.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+module.exports = function(filepath, outpath, requiredir) {
+  var opts = {};
+  if (outpath) {
+    var fs = require('fs');
```

Comment:
Why don't you import module on the top of script? I prefer to do it since it's very clear to see what modules are used in that file.